core/lib/sync/task.ts

```diff
@@ -182,3 +184,37 @@ export function initializeSyncTaskStorage () {
     driver: localForage[config.localForage.defaultDrivers['syncTasks']]
   }))
 }
+
+export function registerSyncTaskProcessor () {
+  const mutex = {}
+  Vue.prototype.$bus.$on('sync/PROCESS_QUEUE', async data => {
```

Comment:
We should make our best effors to avoid using event bus anymore unless it's backwards compatibility related. We've got vuex for that.

Reply:
OK, it's probably an idea for another PR / refactor - EventBus is still quite broadly used across the app. For this purpose - sure it will be pretty easy to move it to the Vuex. Can You please add an issue for that?

Reply:
Sure, #3020